R/run_cluster_analysis.R

```diff
@@ -1,13 +1,18 @@
-#' Load data onto the server associated with
-#' a BiodiverseR::basedata object
+#' Runs a cluster analysis with the given parameters
+#' Uses the given basedata object to call the server
 #'
 #'
 #' @param bd list
```

Comment:
Not sure if this should be a list or something else.  It is an R6 object but that's unlikely to be one of the possible options (and is also not future proof in case we later use a different object system).

Reply:
It seems like bd is of type environment, but that seems like too vague of a definition, so I set it as class basedata for now.